paper/main.bib

```diff
@@ -1,12 +1,26 @@
-
-
 @misc{BickEtal07,
 	author = {I Bicking and B G{\'{a}}bor and {Python Packaging Authority}},
 	howpublished = {\url{https://github.com/pypa/virtualenv}},
 	month = {September},
 	title = {{v}irtualenv: Virtual {P}ython {E}nvironment builder},
 	year = {2007}}
 
+@misc{Chatify,
+	author = {{Chatify~Team}},
+	howpublished = {\url{https://github.com/ContextLab/chatify}},
+	month = {July},
+	title = {Chatify: {A} {J}upyter extension for adding LLM-driven chatbots to interactive notebooks},
+	year = {2023},
+	doi = {10.5281/zenodo.8152315}}
+
+@misc{Neuromatch,
+	author = {{Neuromatch~Team}},
+	howpublished = {\url{https://compneuro.neuromatch.io/}},
+	month = {July},
+	publisher = {Online},
+	title = {{Neuromatch Academy: A free, online, and hands-on computational neuroscience school}},
+	year = {2023}}
```

Comment:
Seems like there's an actual TICS paper to cite for Neuromatch? https://www.sciencedirect.com/science/article/abs/pii/S1364661321000954